dts/arc/quark_se_c1000_ss.dtsi

```diff
@@ -0,0 +1,53 @@
+#include "skeleton.dtsi"
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "snps,arcem4";
+			reg = <1>;
+		};
+	};
+
```

Comment:
We really should have the ARC interrupt controller

Reply:
ok